src/library_signals.js

```diff
@@ -158,6 +158,15 @@ var funs = {
     {{{ makeSetValue('set', 0, 0, 'i32') }}};
     return 0;
   }
+
+  sigwait: function(mask, sig) {
+	// POSIX SIGNALS are not supported, we return -1
+	// to notify users that the function is not implemented
+#if ASSERTIONS
+	err('Calling stub instead of sigwait()');
+#endif
+    return -1;
```

Comment:
The docs say:

```
   On  success, sigwait() returns 0.  On error, it returns a positive error number
       (listed in ERRORS).
```

And apparently the only valid on is `EINVAL` .. so I guess this should be `return {{{ cDefine('EINVAL') }}}`

Reply:
sorry I must have confused it with another sig* function